apigw-s3-terraform/README.md

```diff
@@ -0,0 +1,84 @@
+# REST API as an Amazon S3 proxy in API Gateway
+
+This pattern in CDK offers a boilerplate to generate an Amazon API Gateway REST API endpoint to Amazon S3 with the following methods -
```

Comment:
This talks about CDK?

Reply:
Thanks for the heads up, fixed on new comit